webpack.config.js

```diff
@@ -23,5 +25,18 @@ module.exports = (env) => ({
   },
   plugins: [
     new webpack.DefinePlugin({ 'process.env.API_URL': JSON.stringify(`${env.API_URL}`) }),
+    new CompressionPlugin({
+      filename: '[path].gz[query]',
```

Comment:
Huh so it is literally [path]. For some reason, I thought that meant I had to replace it with the filepath/name. Weirdly worked for me, so I'm not questioning it haha